File: POOUber/PHP/Account.php
<?php

class Accouunt {
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($license, $driver){
        $this->license = $license;
        $this->driver = $driver;
      }
    
      public function PrintDataCar(){
        echo"license: $this->license, conductor: {$this->driver->name}, document: {$this->driver->document}";
      }
}
?>

File: POOUber/Java/Account.java
public class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    public Account(String name, String document) {
        this.name = name;
        this.document = document;
    }

    void printAccountInfo() {
        
        System.out.println("Name → " + name);
        System.out.println("Document → " + document);
        System.out.println("Email → " + email);        

    }

}
